libp2p/standard_setup.nim

```diff
@@ -28,8 +28,9 @@ proc newStandardSwitch*(privKey = none(PrivateKey),
                         triggerSelf = false,
                         gossip = false,
                         secureManagers: openarray[SecureProtocol] = [
+                            # TODO investigate why we're getting fewer peers on public testnets with noise
```

Comment:
This should probably be done at the application level, ie, NBC. From libp2p's standpoint which one gets used first is pretty arbitrary, but for an app (NBC) it might matter and this hides it. I wonder if it should have a default at all?